### ISSUE TITLE
Enhancement: Prevent commercials while allowing bumps for chosen channels

### ISSUE THREAD

**Issue by manodory** [shane-mason/FieldStation42#26]
As some Cable channels don't broadcast commercials nor bumps during a program, this is required. 
Hate to see Married with Children and get multiple commercial breaks (:

**Comment by shane-mason** [comment]
Couple of questions on this one:

- If the content is less than an hour, lets say its 47 minutes long, what do you do you play for the next 13 minutes before the next program start at the top of the hour?
- If the answer is go to the next main content, then that means that the programming wouldn't align with the hour, half hour or even in 15 minute increments - it will end and start at random times depending on how long the main content is? 
- You are thinking of this similar to how a movie channel works?

**Comment by shane-mason** [comment]
In this case, bumps would just be added to the end to pad out the timing?

**Comment by manodory** [comment]
Yes. Our cable TV channels don't have commerials at all, just bumps, and they are played between episodes, movies or live events. 
We had only one channel that was broadcasimg commercials - those were inturrupting the content itself, and a break would start with the channel ident logo short video to indicate a commercial break start, then some commercials, then the commercial-break-end-logo-ident-video, then some bumps and then back to the content. 
Here's an example:
https://youtu.be/Y83GgypNBzY?si=6HjVbum8e7zsFNG4

**Comment by shane-mason** [comment]
Okay - then what this means is the only action that really needs to be enabled here is that the channel only do end-buffering with filler and no cut-in filler. If you leave the commercial folder empty already, then just bumps are will be selected. Maybe a new configuration parameter like this:

`'break_strategy' : 'end'`

to get what you are looking for. If not specified, or set to `inset` to make it work the way it does now.

**Comment by manodory** [comment]
Sounds great!

**Comment by shane-mason** [comment]
Closing as completed.
